### ISSUE TITLE
ns.prompt doesn't support newline in string?

### ISSUE THREAD

**Issue by bandophahita** [danielyxie/bitburner#2520]
Is there a way to get the prompt to support a formatting with newline in the string?  If not, this would be a nice-to-have feature.

```
if !(await ns.prompt(`Are you sure?\nThis will overwrite all the current files`)) {
    ns.exit()
}
```
So the prompt box looks like this:
```
____________________________________________
| Are you sure?                             |
| This will overwrite all the current files |
| Yes                                   No  |
|___________________________________________|
```

**Comment by deathly809** [comment]
Hmm, we dump the string right in as given, not sure if I need to convert to newline, or HTML brs though.

```ts
          <Typography>{prompt.txt}</Typography>
```

Perhaps

**Comment by phyzical** [comment]
another option is just to wrap it in pre tags, though im not sure if there is any side effects..

<img width="735" alt="image" src="https://user-images.githubusercontent.com/5182053/153749591-b1c590e8-a353-484e-958d-25598702b744.png">

**Comment by jacktose** [comment]
It looks like this was accidentally regressed by merging #2880 after #2968. I opened #3171 to re-fix it.
